### PR TITLE
Add AutoTagBot

### PR DESCRIPTION
**Pull request by mnasiadka**

### CODE REVIEW

File: .github/workflows/autotag.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      # This would not be necessary, as AutoTagBot skips already tagged versions
+      # It is however desirable to not trigger this workflow if not required
+      - galaxy.yml
+
+jobs:
+  auto-tag-bot:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
```
Comment (cityofships) [inline .github/workflows/autotag.yml:17]: We'd probably want `v4` here

File: .github/workflows/autotag.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+      - name: Release new versions using AutoTagBot
+        uses: snapserv/action-autotagbot@master
```
Comment (markgoddard) [inline .github/workflows/autotag.yml:20]: This doesn't seem like a very popular action. Is it risky?

File: .github/workflows/autotag.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        with:
+          api_token: '${{ secrets.GITHUB_TOKEN }}'
```
Comment (markgoddard) [inline .github/workflows/autotag.yml:22]: Should we be wary of handing our token to a 3rd party action?

**Review by markgoddard — COMMENTED** [review]
If we do go down this route, it would be good to standardise across our collections. We have stuff in release train to roll out standard workflows to repos.

**Comment by mnasiadka** [comment]
And I just wanted to check how it works - since when do we comment on draft PRs? 😀 
We could write our own script instead of using that action, I haven't found any other generic action we could use.

**Comment by markgoddard** [comment]
> And I just wanted to check how it works - since when do we comment on draft PRs? 😀 We could write our own script instead of using that action, I haven't found any other generic action we could use.

I comment on draft PRs on high-level things, but try to ignore the details.

**Comment by mnasiadka** [comment]
> > And I just wanted to check how it works - since when do we comment on draft PRs? 😀 We could write our own script instead of using that action, I haven't found any other generic action we could use.
> 
> I comment on draft PRs on high-level things, but try to ignore the details.

Just joking - I think I'd prefer to have a workflow that watches galaxy.yaml for changes instead of pushing tags or ,,clicking'' through GH releases - but I'll analyse the security and might be we'd like to have our own workflow for that.

**Pull request closed without merging**